docs/tutorials/developer-journey/level-1/1.3-first-dapp.md

```diff
@@ -0,0 +1,723 @@
+├── README.md
+├── dfx.json           # The configuration file for your Internet Computer dapp
+├── package-lock.json  # Node.js package configuration
+├── package.json       # Node.js package configuration
+├── src
+│   ├── poll_backend   # Folder containing the source code of your dapp's backend.
+│   │   └── main.mo    # The default source code file; this is what we'll primarily be working with in this tutorial.
+│   └── poll_frontend  # Folder containing the frontend configuration and assets of your dapp. 
+│       └── ...
+└── webpack.config.js  # Web dapp bundler configuration.
+```
+
+You will notice that the file structure above has two sub-directories within `src`. That is because when you created the project `poll` via `dfx new poll`, the project by default created **two** canisters that talk to each other:
+- `src/poll_backend` which contains the files that serve as the backend of the dapp.
+- `src/poll_frontend` which contains the files that serve as the frontend of the dapp.
+
+This file structure is the default template `dfx` creates for a new project. If you'd like to create a project without a frontend canister, the `--no-frontend` flag can be used with the `dfx new` command. 
+
+## Writing the backend canister code
+
+Let's start writing the code for the backend of our dapp. Open your code editor of choice, then open the file `src/poll_backend/main.mo`. By default, this file will contain the same code we reviewed in the [introduction to dfx](../level-0/06-intro-dfx.md) module. Since we want to write our own code, delete the entire contents of this file. 
+
+### Creating an actor 
+
+First, we'll create an actor. Recall that an actor is a process with an encapsulated state. Actors contain both code and data, and communicate by sending and receiving messages. A canister can only contain a single actor. 
+
+In your `src/poll_backend/main.mo` file, paste the following code:
+
+:::caution
+The following example is a **code snippet** that is part of a larger code file. This snippet may return an error if run on its own. To view the full code file that should be run, please see [final code](#final-code).
+:::
+
+```motoko
+actor {
+
+    //actor code goes here
+    
+}
+```
+
+This code defines an actor, but it is currently empty. It does not define any data or send and receive messages. 
+
+Next, let's add some code inside the actor that proposes a question for our poll app. For example, we can ask the user, "What is your favorite programming language?". 
+
+To add this question, insert the following code inside of the actor definition:
+
+:::caution
+The following example is a **code snippet** that is part of a larger code file. This snippet may return an error if run on its own. To view the full code file that should be run, please see [final code](#final-code).
+:::
+
+```motoko
+    var question: Text = "What is your favorite programming language?";
+```
+
+:::info
+#### What does this code do?
```

Comment:
it looks like having the header inside the admonition breaks the formatting of the admonition. see screenshot.
<img width="636" alt="Screenshot 2023-08-24 at 4 29 29 PM" src="https://github.com/dfinity/portal/assets/98767015/3d08f7b0-1ff6-4b60-8f27-74f77e670514">